File: TourApp/data/src/main/java/io/github/pedrofraca/data/datasource/database/StageDatabaseFactory.java
package io.github.pedrofraca.data.datasource.database;

import android.content.Context;

import androidx.room.Room;

public class StageDatabaseFactory {

    public static StageDatabase getDatabase(Context context){
        return Room.databaseBuilder(context, StageDatabase.class, "db-contacts")
                .allowMainThreadQueries()
                .fallbackToDestructiveMigration()
                .build();
    }

}

File: TourApp/data/src/main/java/io/github/pedrofraca/data/datasource/database/StageDatabaseFactory.kt
package io.github.pedrofraca.data.datasource.database

import android.content.Context
import androidx.room.Room

object StageDatabaseFactory {
    fun getDatabase(context: Context?): StageDatabase {
        return Room.databaseBuilder(context!!, StageDatabase::class.java, "tour-db")
                .fallbackToDestructiveMigration()
                .build()
    }
}